lib/sanity-queries.ts

```diff
@@ -57,6 +57,28 @@ export const aboutPageQuery = groq`
   }
 `;
 
+export const faqPageQuery = groq`
+*[_type == 'faqPage'][0] {
+  ...,
+  title,
+  paragraph,
+  faqList[] {
+    ...,
+    faq {
```

Comment:
@alexxuss1996 I notice that `faq` is returning `null` in the response you showed me before, this happens because there's no a `faq` property within `faqList` as you can [see here](https://juguetear-web.vercel.app/studio/production/desk/preguntasYRespuestasFrequentes%2Cinspect%3Don):

```json
"faqList": [
    {
      "_key": "611f18c93681",
      "_type": "faq",
      "answer": [
        {
          "_key": "9656efacd81d",
          "_type": "block",
          "children": [
            {
              "_key": "d6dc3b31da3d0",
              "_type": "span",
              "marks": [],
              "text": "El proyecto Juguetear..."
            }
          ],
          "markDefs": [],
          "style": "normal"
        }
      ],
      "question": "¿Qué ingresos tienen, y cómo se mantienen?"
    },
  ]
 ```
 
 And actually, is no really necessary to be this "verbose" for this case, since:
 ```
   faqList[] {
    ..., <= This already means "give me every property".
    faq { <= 'faq' can be removed here an is should still work nicel, just like you showed me.
```

In fact, in this case, you can simply do `*[_type == 'faqPage'][0]` and that will bring all we need! :smile:
Although is good to know how these queries work, nice work! :rocket: 

Approving! :shipit:

Reply:
Ha, that's useful to know, Thanks. Although I tried groq for the first time, that's why it's verbose 😁

Reply:
> Huh, that's useful to know, Thanks. Although I tried groq for the first time, that's why it's verbose 😁

For me, it helps to think about like this, when you use `{ property }` is similar to [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) in JavaScript, and the `...` will be like [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_object_literals), however, in GROQ queries they called it [ellipsis operator](https://www.sanity.io/docs/how-queries-work#8d57d7b0dccb).

I hope that helps!